README.md

```diff
@@ -83,13 +83,74 @@ pip install typer requests PyGithub rich pydantic
 
 ## Authentication
 
-You need a GitHub personal access token with appropriate permissions:
+### Quick Start
 
-1. Go to GitHub Settings → Developer settings → Personal access tokens
-2. Create a token with these scopes:
-   - `repo` (for private repositories)
-   - `public_repo` (for public repositories)
-   - `read:org` (to list organization repositories)
+**You need a GitHub personal access token with these permissions:**
+
+- **Organization access**: Read organization repositories
+- **Repository access**: Read pull requests, contents, metadata, and checks
+- **Pull request management**: Write access to approve and merge pull requests
+
+### Token Types
+
+GitHub offers two types of personal access tokens: **fine-grained**
+(recommended) and **classic**.
+
+### Fine-Grained Personal Access Tokens (Recommended)
+
+Fine-grained tokens provide more precise control and better security.
+Create one at:
+GitHub Settings → Developer settings → Personal access tokens → Fine-grained tokens
+
+#### Required Repository Permissions
+
+Your token needs these permissions on **all repositories** in the target organization:
+
+- **Pull requests**: Read and Write
+  - Read: View PR details, status, files, commits, and reviews
+    (for finding blocked PRs and similarity matching)
+  - Write: Approve and merge pull requests (for automated bulk operations)
+- **Contents**: Read
+  - Read repository files and commit information (for analyzing file changes in PRs)
+- **Metadata**: Read
+  - Basic repository information and access (required for all GitHub API operations)
+- **Checks**: Read
```

Comment:
This permission is not available to a PAT currently. The only way to get all permissions needed is a Legacy token.